admin/class-yoast-notification-center.php

```diff
@@ -96,7 +96,12 @@ public static function is_notification_dismissed( Yoast_Notification $notificati
 		$user_id       = ( ! is_null( $user_id ) ? $user_id : get_current_user_id() );
 		$dismissal_key = $notification->get_dismissal_key();
 
-		$current_value = get_user_meta( $user_id, $dismissal_key, true );
+		$current_value = get_user_option( $dismissal_key, $user_id );
+
+		// Back-compat.
```

Comment:
Please be a bit more descriptive in your comments. Also refrain from abbreviating words.